src/Fhir.Anonymizer.Tool/configuration-sample.json

```diff
@@ -11,6 +11,8 @@
     {"path": "nodesByType('Address').country", "method": "keep"},
     {"path": "nodesByType('Address').state", "method": "keep"},
 
+    {"path": "Resource.id", "method": "cryptoHash"},
+    {"path": "Bundle.entry.fullUrl", "method": "redact"},
```

Comment:
It seems we use different method for this Bundle.entry.fullUrl. Is this by design or both cryptoHash and redact are OK?

Reply:
Thanks. Full url contains resource id. But considering fullUrl will no longer work if we just replace id partly, I think we'd better redact or cryptoHash whole here. This is my current personal view and we can refine the logic in the future.